packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -1117,6 +1093,58 @@ export default async function initSanity(
     })
   }
 
+  async function updateProjectCliInitializedMetadata() {
```

Comment:
These were defined in an array of promises. I've just defined them further down in the code as functions, but still call them in the same place as we did previously.